static/docs/user-guide/dvcignore.md

```diff
@@ -38,7 +38,107 @@ by design works similar way as `.gitignore` does.
 
 The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
 
-## Example
+## Examples: Modification of ignored data
+
+Lets see if what happens when we modify ignored file.
+
+```dvc
+$ mkdir data
+$ echo data1 >> data/data1
+$ echo data2 >> data/data2
+$ tree .
+
+.
+└── data
+    ├── data1
+    └── data2
+```
+
+We created data directory. Lets ignore part of data and add `data` under DVC
+control.
+
+```dvc
+$ echo data/data1 >> .dvcignore
+$ cat .dvcignore
+
+data/data1
+
+$ dvc add data
+$ tree .dvc/cache
+
+.dvc/cache
+├── 54
+│   └── 40cb5e4c57ab54af68127492334a23.dir
+└── ed
+    └── c3d3797971f12c7f5e1d106dd5cee2
+```
+
+As we can see, `data1` has been ignored. Cache contains only one file entry
+(for `data2`) and one dir entry (`data`).
+
+Now, lets modify data1 and see if it affects `dvc status`.
```

Comment:
again `data1`

Reply:
Also `lets` -> `let's`. Fixed in c8591d4